src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -28,6 +29,18 @@ class AboutCommand extends ContainerAwareCommand
 {
     /**
      * {@inheritdoc}
+     *
+     * @deprecated since version 3.4, to be removed in 4.0
+     */
+    protected function getContainer()
```

Comment:
just wondering, about all those getContainer() method added for BC layer:
shouldn't we instead make all commands `@final since version 3.4`? Wouldn't that be enough?